packages/internal/simple-store/src/simple-store.ts

```diff
@@ -1,15 +1,15 @@
-export type Awaitable<V> = V | PromiseLike<V>
+import { Awaitable } from './util.js'
 
-export type Key = string | number
+export type Key = NonNullable<unknown>
```

Comment:
I had a question about this change above in the changesets, just noting again here for good measure.